upload/docker_images/validator/validator_harness.py

```diff
@@ -82,6 +82,10 @@ def _stage_files_to_be_validated(self):
                                                                           file_path=staged_file_path))
             staged_file_path.parent.mkdir(parents=True, exist_ok=True)
             self._download_file_from_bucket_to_filesystem(s3_bucket_name, s3_object_key, staged_file_path)
+            if not staged_file_path.is_file():
+                raise UploadException(status=500, title="Staged file path is not a file",
+                                      detail=f"Attempting to stage file path {staged_file_path} failed because it is "
+                                      f"not a file.")
             if not staged_file_path.is_file():
```

Comment:
If we are now invoking an UploadException for `if not staged_file_path.is_file()` can't this second if statement that is raising a TryAgain be removed?

Reply:
Oops.. just forgot to delete that.

Reply:
Done.